python/tests/benchmark_variable.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
+# @file
+import time
+import scipp as sc
+from scipp import Dim
+import numpy as np
+
+
+def assign_from_numpy_1d(size):
+    a = np.ndarray(dtype=np.float64, shape=(size, ))
+    a.fill(0)
```

Comment:
Why not use `np.zeros()`?

Reply:
Interesting. Using `zeros` gives the same oddly high performance I saw earlier.

Reply:
😨

Reply:
What about `ones` or `arange`?

Reply:
`ones` is the same performance as the create then fill I had above.